Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.py

```diff
@@ -39,9 +39,57 @@ def main():
             'mapping': {
                 'description': 'malwarefamily'
             }
+        },
+        'https://faf.bambenekconsulting.com/feeds/dga-feed.gz': {
+            'fieldnames': ['value', 'description',
+                           'date_created',
+                           'info'],
```

Comment:
Why aren't we mapping all these other fields and only description? Off the top of my head, `date_created` could probably map to `firstseenbysource`. Please consult with @David-BMS - we should be mapping as many fields here as possible (this comment applies for all the new feeds you added here to Bambenek).

Reply:
@avidan-H We would like to use @David-BMS mapping for the old feeds as well?

Reply:
Yeah, speak to @David-BMS - we should be mapping as much data as possible in the code.